src/DataManagement/Entries/EntryManager.cs

```diff
@@ -1,109 +1,108 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
 using ePiggyWeb.DataBase;
-using ePiggyWeb.DataBase.Models;
 using ePiggyWeb.Utilities;
-using Microsoft.EntityFrameworkCore;
+using IListExtension;
 
-namespace ePiggyWeb.DataManagement
+namespace ePiggyWeb.DataManagement.Entries
 {
-    public class EntryManager
+    public class EntryManager : IEntryManager
```

Comment:
So EntryManager (and by extension GoalManager) basically takeover the same functionality as old Data.cs class used to do.
EntryManager reflects all the change we do on a local list to the database as well. In other words, if we want to add a entry, we call the EntryManager.Add method, which will first update the database, and if the database was successfully updated, it will update the local list as well, and analogously for every other method in EntryManager.
Also EntryManager has the duty of reading the data from the database on it's creation.